src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/ProjectModifier.cs

```diff
@@ -48,8 +51,7 @@ public async Task AddAuthCodeAsync()
                 if (csProjFiles.Count() != 1)
                 {
                     var errorMsg = string.Format(Resources.ProjectPathError, _toolOptions.ProjectFilePath);
-                    _consoleLogger.LogJsonMessage(new JsonResponse(Commands.UPDATE_PROJECT_COMMAND, State.Fail, errorMsg));
-                    _consoleLogger.LogMessage(errorMsg, LogMessageType.Error);
+                    _consoleLogger.LogJsonMessage(new JsonResponse(Commands.UPDATE_PROJECT_COMMAND, State.Fail, null, errorMsg)); // TODO: every time LogJsonMessage is called, it should also log message if --json is disabled.
```

Comment:
if we are passing `null`, for readability, let's use `parameterName : null` to know which parameter is the null one.

Reply:
I would edit all instances of the `new JsonResponse()`